hackerrank/src/Algorithms/Strings/Anagram.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Strings.Anagram;
+
+public class Result
+{
+    public static int anagram(string s)
```

Comment:
```suggestion
    public static int Anagram(string s)
```
Не пішы з маленькай літары! У СіШарп усе назвы звычайна пычынаецца з вялікай літары, больш вядома як PascalCase